src/transformers/generation_tf_utils.py

```diff
@@ -1366,6 +1370,9 @@ def _generate(
             use_cache (`bool`, *optional*, defaults to `True`):
                 Whether or not the model should use the past last key/values attentions (if applicable to the model) to
                 speed up decoding.
+            seed (`List[int]`, *optional*):
```

Comment:
Is it a list of ints or just int? Or both? Don't think the type hint is correct here